### PR TITLE
santoshi

### PR DESCRIPTION
**Pull request by santoshinawkhare**

### CODE REVIEW

**Review by pradip0852 — APPROVED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
never use h1 tag in head tag
add new line at end in every file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
1. follow HTML rules
2. add new line at the end

File: assignment5.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                <li>English</li>
+                <li>Hindi</li>
+                <li>Marathi </li>
+            </ul>
+            <h2 style="color:rgb(14, 154, 247) ;">Personal Information</h2><hr>
+            <ul>
+                <li>Name: - SANTOSHI ASHOK NAWKHARE</li>
+                <li>DOB: - 5/08/1998</li>
+                <li>Father’s Name: -Ashok Raghunath Nawkhare</li>
+                <li>Cooking, Reading. </li>
```
Comment (Pawan1005) [inline assignment5.html:58]: add hobbies there

File: assignment5.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            </ul>
+            <br>
+            <br>
+            <b>Date:</b>
+            <br>
+            <br>
+            <b>Place: Nagpur</b>
+        </body>
+</html>
```
Comment (Pawan1005) [inline assignment5.html:67]: add new line here

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by pradip0852 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**